src/test/direct_messenger/test_direct_messenger.cc

```diff
@@ -11,8 +11,8 @@
 #include "common/ceph_argparse.h"
 
 #include "DirectMessenger.h"
-#include "msg/FastStrategy.h"
-#include "msg/QueueStrategy.h"
+#include "FastStrategy.h"
+#include "QueueStrategy.h"
```

Comment:
might want to update `src/msg/CMakeLists.txt` and `src/test/direct_messenger/CMakeLists.txt` accordingly.

Reply:
I have made some changes, but direct messenger don’t seem to work properly even if uncomment cmake statements.(It was commented out long ago by sage at d290ffc098 due to no users and failing). Maybe we can cleanup direct_messenger in the future.

Reply:
@Rethan sure. what i really wanted was to remove `QueueStrategy.cc` from `msg/CMakeLists.txt`. and it's addressed in your latest revision. so all good!